ml/requirements.txt

```diff
@@ -0,0 +1,11 @@
+scikit-learn
+matplotlib
```

Comment:
Interesting, maybe we add a few things to https://github.com/graphistry/pygraphistry/blob/53bd7a779b9efc216301fcee94b493de9184cbc2/setup.py#L16

like `ml` (umap, ...) + `ai` (dgl)